README.md

```diff
@@ -1,68 +1,95 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Could you organize the endpoints within the README using a table? You can reference this [resource](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables) for table syntax. 

Adopt the structure for the README [here](https://github.com/andela/engineering-playbook/tree/master/5.%20Developing/Conventions#message-body)

Reply:
Yes, I will do that

Reply:
@KicaRonaldOkello Correct the spelling of `endpoints` in the table.